chore: dockerize laravel-example (#4)
momentohq/laravel-example: README.md

## Run the app via Docker

Before building a Docker image, create `.env` file in `docker` directory with the following env variables:
- `MOMENTO_AUTH_TOKEN`=<YOUR_AUTH_TOKEN>
- `MOMENTO_CACHE_NAME`=<YOUR_CACHE_NAME>
- `WEATHER_API_KEY` this is for weather API. Check out [OpenWeather](https://openweathermap.org/) to get an API key.

Build a Docker image for the app:
```bash
cd docker
docker build --tag laravel-example .
```
If you run into the API limits from GitHub, add a GitHub personal access token to your composer configuration:
```bash
export COMPOSER_AUTH=<YOUR_GITHUB_ACCESS_TOKEN>
docker build --tag laravel-example --build-arg COMPOSER_AUTH=$COMPOSER_AUTH .

> ```

Reviewer: Somewhere in here I think we should add a note that the build involves compiling the grpc extension and that it should be expected to take a really long time. I know we were both suspicious of how long it was taking, and I'd like to address that upfront in the instructions.

Author: Good call.

## Run the app via Docker

Before building a Docker image, create `.env` file in `docker` directory with the following env variables:
- `MOMENTO_AUTH_TOKEN`=<YOUR_AUTH_TOKEN>
- `MOMENTO_CACHE_NAME`=<YOUR_CACHE_NAME>
- `WEATHER_API_KEY` this is for weather API. Check out [OpenWeather](https://openweathermap.org/) to get an API key.

Build a Docker image for the app:
```bash
cd docker
docker build --tag laravel-example .
```

> **Notes**: Building the `laravel-example` image involves compiling the PHP gRPC extension.
> And this step to install gRPC is expected to take a long time to complete during build when you first build an image.

Reviewer: ```suggestion
**Note**: Building the `laravel-example` image involves compiling the PHP gRPC extension, which will take several minutes to complete.
```

Author: I like your concise version 👍
